### PR TITLE
SSB4: Add Volco

### PR DESCRIPTION
**Pull request by Volco**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -1972,6 +1972,40 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+			}
+		},
+		onFoeModifyMove(move, pokemon) {
+			if (move.thawsTarget) {
+				delete move.thawsTarget;
+			}
+			if (move.flags["defrost"]) {
+				delete move.flags["defrost"];
+			}
+		},
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts]: Couldn't you merge this to `onAnyModifyMove`?

File: data/mods/ssb/conditions.ts [hunk 2]
```diff
@@ -1737,6 +1737,25 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
...
+		onFaint(source, target, effect) {
+			if (effect.id === 'glitchexploiting') {
+				this.add(`c|${getName('Volco')}|Dammit, time for a reset.`);
+				return;
+			}
+			this.add(`c|${getName('Volco')}|Looks like the game fro-`);
+			this.add(`raw|<div class="broadcast-red">` +
+				`<strong>This Pokemon Showdown battle has froze!</strong><br />Don't worry, we're working on fixing it so just carry on like you never saw this.` +
+				`</div>`);
+		},
```
Comment (KrisXV) [inline data/mods/ssb/conditions.ts]:
```suggestion
			this.add(`raw|<div class="broadcast-red"><strong>This Pokemon Showdown battle has frozen!</strong><br />Don't worry, we're working on fixing it, so just carry on like you never saw this.</div>`);
		},
```

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -4675,6 +4675,76 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+				target.forceSwitchFlag = true;
+				source.forceSwitchFlag = true;
+			}
+		},
+		secondaries: [
+			{
+				chance: 20,
+				onHit(target, source) {
+					if (this.randomChance(1, 10)) {
+						const status = ['frz', 'par'][this.random(2)];
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						const status = this.sample(['frz', 'par']);
```

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -4675,6 +4675,76 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+					} else {
+						target.trySetStatus('brn');
+					}
+				},
+			},
+			{
+				chance: 30,
+				onHit(target, source) {
+					if (this.randomChance(1, 10)) {
+						const status = ['frz', 'par'][this.random(2)];
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						const status = this.sample(['frz', 'par']);
```

File: data/mods/ssb/abilities.ts (hunk 1 above)
Comment (Volco) [reply data/mods/ssb/abilities.ts]: i forgot that existed, done

**Pull request merged**